index.html

```diff
@@ -2344,35 +2344,62 @@ <h2>
         A {{Window/window}} has an associated {{LaunchParams}} <dfn>unconsumed
         launch params</dfn>.
       </p>
-      <p>
-        When a [=file type=] is registered with a web app and one or more of
-        these registered files are launched on the platform run the steps to
-        [=execute a file handler launch=], given the launched {{FrozenArray}}
-        of {{FileSystemHandle}}s |files:FrozenArray&lt;FileSystemHandle&gt;|
-        and the corresponding [=file handler|file_handler=].
-      </p>
       <p>
         The steps to <dfn>execute a file handler launch</dfn> are given by the
-        following algorithm. The algorithm takes
-        |files:FrozenArray&lt;FileSystemHandle&gt;| and a [=file handler=]
-        |file_handler:file handler|.
+        following algorithm. The algorithm takes [=list=] |files:list| and a
+        [=ordered map=] |manifest:ordered_map| which holds results from
```

Comment:
you could cut out a step here and say it takes a process manfest - or instead of [=process the `file_handler` member=] it could just be [=process the manifest=][

Reply:
done